WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowSettingsViewController.swift

```diff
@@ -198,6 +198,8 @@ class BloggingRemindersFlowSettingsViewController: UIViewController {
         configureConstraints()
         populateCalendarDays()
         refreshNextButton()
+
+        navigationController?.setNavigationBarHidden(true, animated: false)
```

Comment:
Nitpicky: IMHO this is an intrinsic property of our `BloggingRemindersNavigationController` and setting it here isn't great.  My reasoning is that this VC could be shown in the future in another navigation controller that doesn't hide its bar.

My suggestion would be to move this call, and the same call in `BloggingRemindersFlowIntroViewController:106` to somewhere inside `BloggingRemindersNavigationController` (either the `init` method or `viewWillAppear` / `viewDidAppear`).

Reply:
Good catch!!